htdocs/compta/facture/card.php

```diff
@@ -1197,7 +1197,7 @@
 									// Don't add lines with qty 0 when coming from a shipment including all order lines
 									if($srcobject->element == 'shipping' && $conf->global->SHIPMENT_GETS_ALL_ORDER_PRODUCTS && $lines[$i]->qty == 0) continue;
 									// Don't add closed lines when coming from a contract
-									if($srcobject->element == 'contrat' && $lines[$i]->statut == 5) continue;
+									if($srcobject->element == 'contrat' && $lines[$i]->statut != 4) continue;
```

Comment:
If service was just suspended, i think a lot of use case will need to invoice despite that.
I suggest a hidden option to be able to choose status to invoice 

if (! isset( $conf->global->CONTRACT_EXCLUDE_SERVICES_STATUS_FOR_INVOICE))
 $conf->global->CONTRACT_EXCLUDE_SERVICES_STATUS_FOR_INVOICE = '0,5';

and replace test with

&& in_array($lines[$i]->statut, explode(',', $conf->global->CONTRACT_EXCLUDE_SERVICES_STATUS_FOR_INVOICE))